cobra/test/test_io/test_sbml.py

```diff
@@ -260,15 +261,16 @@ def test_missing_flux_bounds1(data_directory):
         assert r1.upper_bound == config.upper_bound
 
 
+# FIXME
```

Comment:
What needs to be fixed here?

Reply:
Was a failing test, just forgot to remove comment after fixing things.